### PR TITLE
[WIP] gnome: update to 3.34

### PR DESCRIPTION
**Pull request by Wychmire**
Working on updating the GNOME stuff.

Feel free to cancel the travis ci whenever (if possible) since I don't need to anything to build rn

### CODE REVIEW

**Comment by Hoshpak** [comment]
`gnome-tweaks: update to 3.33.90`

this is an unstable development version, please only use stable releases.

**Comment by Wychmire** [comment]
> `gnome-tweaks: update to 3.33.90`
> 
> this is an unstable development version, please only use stable releases.

Whoops. Should be fixed now!

**Comment by Wychmire** [comment]
@Gottox Gnome packages are pretty much maintained by you, mind taking a look at what I've done?

Also, if anyone has any ideas why the merge status thing says "This branch cannot be rebased due to conflicts" I'd love to hear them. As far as I can tell there aren't any conflicts and rebasing works fine on my end.

File: srcpkgs/atkmm/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'atkmm'
 pkgname=atkmm
-version=2.28.0
+version=2.29.1
```
Comment (Gottox) [inline srcpkgs/atkmm/template]: Unstable version

File: srcpkgs/gcr/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'gcr'
 pkgname=gcr
-version=3.28.1
-revision=3
+version=3.33.4
+revision=1
```
Comment (Gottox) [inline srcpkgs/gcr/template]: unstable

File: srcpkgs/atkmm/template (hunk 1 above)
Comment (Wychmire) [reply srcpkgs/atkmm/template]: Fixed.

File: srcpkgs/gcr/template (hunk 2 above)
Comment (Wychmire) [reply srcpkgs/gcr/template]: Fixed.

**Comment by Wychmire** [comment]
> I've marked all unstable versions I found. Did you test the gnome update?

Not yet. I was testing #14656 which messed something up on my main computer. I've started building all the gnome stuff on a different computer which should work.

**Comment by Gottox** [comment]
I'm doing a test build currently.

**Comment by Gottox** [comment]
gobject-introspection needs this: I'm not sure if `build_helper` is needed here at all: https://github.com/Gottox/void-packages/commit/f583263e0046ffadc6d0602bcdb93d0f9569cd19

**Comment by Wychmire** [comment]
> gobject-introspection needs this: [Gottox@f583263](https://github.com/Gottox/void-packages/commit/f583263e0046ffadc6d0602bcdb93d0f9569cd19)

Made the marked changes.

Built and updated gnome 3.34 locally, everything seems to be working except the required packages for the gnome-apps and gnome-games-collection weren't built when I built the main gnome package. Gnome itself is working great.

**Comment by Wychmire** [comment]
> You did missed one shlib change of evolution-data-server and didn't remove a outdated one.

Whoops. Fixed!

**Comment by Johnnynator** [comment]
Mutter fails to compile on x86_64-musl for me (didn't test anywhere else)
http://ix.io/1YdD

**Comment by Wychmire** [comment]
> Mutter fails to compile on x86_64-musl for me (didn't test anywhere else)

Should be fixed now, thanks for pointing that out!

**Comment by shizonic** [comment]
Merge conflicts? Would appreciate a merge of gnome 3.34 ...

@Wychmire Thx for your work !!

**Comment by Wychmire** [comment]
I'm working on splitting up this PR into smaller pieces so it's easier to merge and hopefully I can figure where the conflict is, since Github won't tell me.

Some of the things are out of date (3.34.1 is already out) so while I rework it I'll be re-updating those applications. Hopefully it gets merged soon!

**Pull request closed without merging**